.circleci/config.yml

```diff
@@ -79,6 +79,16 @@ jobs:
         path: gl-container-scanning-report.json
         destination: security-scan
 
+  release:
+    executor: default
+    steps:
+      - checkout
+      - run:
+          command: |
+            sudo apt update && sudo apt install hub git
+            GO111MODULE=on go get k8s.io/release/cmd/release-notes
```

Comment:
I think you don't need `GO111MODULE` as the image is golang and will run under the golang paths

Reply:
also for all the non-make commands just add them in a separate `Makefile` command